locallibrary/catalog/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    
+]
```

Comment:
Khai báo empty list thì viết gọn như sau e nhé
```suggestion
urlpatterns = []
```